python/cudf/cudf/core/column/column.py

```diff
@@ -1167,6 +1167,9 @@ def corr(self, other: ColumnBase):
     def nans_to_nulls(self: T) -> T:
         return self
 
+    def can_fillna(self) -> bool:
```

Comment:
I find this name a little confusing because you "can" fillna a column that contains no nulls, it's just a no-op. Maybe something like `contains_fillable_entries` would be clearer? I don't want to make it too long, but would prefer something less ambiguous.

Reply:
Since `nan_count` and `null_count` are properties, maybe this should be a property too?

Reply:
Changed it to `contains_na_entries` property